airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/catalog.py

```diff
@@ -139,6 +171,16 @@ def oss_catalog_from_metadata(catalog_derived_metadata_definitions: List[Partial
     return construct_catalog_from_metadata(catalog_derived_metadata_definitions, "oss")
 
 
+@asset(group_name=GROUP_NAME)
+def cloud_catalog_from_metadata_and_spec(cloud_catalog_from_metadata: dict, cached_specs: OutputDataFrame) -> dict:
```

Comment:
I'm thinking we should merge `*_catalog_from_metadata_and_spec` into `*_catalog_from_metadata`